temporal_sqlalchemy/tests/test_temporal_models.py

```diff
@@ -316,17 +317,16 @@ def test_disallow_flushes_within_clock_ticks_when_strict(self, session, session_
             t.prop_a = 2
 
             with pytest.raises(AssertionError) as excinfo:
-                eval('session.{func_name}()'.format(func_name=session_func_name))
+                eval('session.{func_name}()'.format(func_name=session_func_name))  # pylint: disable=eval-used
```

Comment:
Instead of using `eval` and string interpolation, could we use `getattr(session, session_fun_name)()`? Here and throughout.